gapis/api/gles/compat.go

```diff
@@ -209,7 +209,7 @@ func compat(ctx context.Context, device *device.Instance, onError onCompatError)
 			// TODO(dsrbecky): This might make some commands valid for replay which were invalid on trace.
 			scs := FindStaticContextState(s.Arena, cmd.Extras())
 			if !scs.IsNil() && !version.IsES && scs.Constants().MajorVersion() < 3 {
-				clone := cmd.Clone(s.Arena)
+				clone := cmd.Clone(s.Arena).(*EglMakeCurrent)
```

Comment:
Consider adding a `clone(arena.Arena) *T` method (note the lowercase), which `Clone(arena.Arena) api.Cmd` delegates on to.